source/modules/overview/actions.rst

```diff
@@ -0,0 +1,178 @@
+Actions
+-------
+
+The different actions that are available on an object depend on the permissions assigned in user profile. 
+
+Likewise, some actions in the tab depend on profile permission, such as mass actions.
+
+This page describes only main actions that can apply to different types of objects. If an action is specific to an object, it will be described in the documentation of this object.
+
+Creation
+^^^^^^^^
+
+The permission to create an object depend on defined permission in user profile.
+
+For all inventory objects, creation is realized in the same way:
+
+#.  Connect to GLPI ;
+#.  Go to the object page, for example `Assets > Computers` to add a computer ;
+#.  Click on the "+" button located in horizontal menu ;
+#.  If the inventory object has a template management, choose the template to be applied ; 
+#.  Fill in the different fields of the empty form ;
+#.  Validate.
+
+An option located in user preferences, *After creation, go to created element*, allows to choose, after creating an object, if the newly created object is displayed or if a new empty form is displayed in order to create another object.
+
+Modification
+^^^^^^^^^^^^
+
+The permission to modify an object depend on defined permission in user profile.
+
+For all inventory objects, creation is realized in the same way:
+
+#.  Connect to GLPI ;
+#.  Go to the object page, for example, to modify a computer, go to `Assets > Computers` ;
+#.  Or alternatively: search the object in the search engine available on top of objects list ;
+#.  Two choices of modification are then possible:
+
+   * Unitary modification:
+
+      #.  Click on object name ;
+      #.  Modify fields ;
+      #.  Click on button **Update**.
+
+   * Mass modification:
+
+      #.  Select the checkbox located on the left of the object name ;
+      #.  Click on button **Actions** ;
+      #.  Choose field to be modified ;
+      #.  Enter new field value ;
+      #.  Click on button **Post**.
+
+Display
+^^^^^^^
+
+The permission to display an object depend on `read` permission in user profile.
+
+If the permission to display an object is not granted, the name of this object will not appear in GLPI different menus. For example, if the permission to read a `Computer` type object is not granted in the user's profile, the sub-menu `Computer` will not appear in menu `Assets`, even if the permission to modify a `Computer` is granted.
+
+For all inventory objects, displaying an object is realized in the same way:
+
+#. Connect to GLPI ;
+#. Go to the object page, for example, to display a computer, go to `Assets > Computers` ;
+#. Or alternatively: search the object in the search engine available on top of objects list ;
+#. Click on object name..
+
+
+Attaching a document
+^^^^^^^^^^^^^^^^^^^^
+
+For all inventory objects, attaching a document is realized in the same way:
```

Comment:
Not only inventory objects as far as I know

Reply:
Then "for all objects"?

Reply:
There are a few non-inventory types that can have Documents attached.
The current list of document types:
```php
$CFG_GLPI["document_types"] = ['Budget', 'CartridgeItem', 'Change', 'Computer',
                                                  'ConsumableItem', 'Contact', 'Contract',
                                                  'Document', 'Entity', 'KnowbaseItem', 'Monitor',
                                                  'NetworkEquipment', 'Peripheral', 'Phone',
                                                  'Printer', 'Problem', 'Project', 'ProjectTask',
                                                  'Reminder', 'Software', 'Line',
                                                  'SoftwareLicense', 'Supplier', 'Ticket','User',
                                                  'Certificate', 'Cluster', 'ITILFollowup', 'ITILSolution',
                                                   'ChangeTask', 'ProblemTask', 'TicketTask', 'Appliance'];
```

Reply:
I  propose:
"When available for the object type, attaching a document is realized...."

Reply:
s/object type/item type/

Reply:
Agree

Reply:
Changed